Model/Game.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace GameOfLife
+{
+    [Serializable]
+    public class Game
+    {
+        /// <summary>
+        /// Properties.
+        /// </summary>
+        public int Height { get; set; }
+        public int Width { get; set; }
+        public int[,] currentGeneration { get; private set; }
+        public int iterationCount = 1;
+
+        /// <summary>
+        /// Constructor without parameters.
+        /// </summary>
+        public Game()
+        {
+            Height = 16;
+            Width = 16;
+            currentGeneration = new int[Height, Width];
+
+            InitializeBoard();
+        }
+
+        /// <summary>
+        /// Constructor with parameters from user.
+        /// </summary>
+        /// <param name="x Number of rows (height)."></param> 
+        /// <param name="y  Number of columns (width)."></param>
+        public Game(int x, int y)
```

Comment:
One letter variables sometimes are OK as iteration variables in for statements, but not as method variables.
In general you should use more descriptive names. In your case you already now them:
x Number of rows (height) => numberOfRows. The same for columns.
Descriptive names makes code more readable for other developers.

Reply:
Variable names changed to rows and columns.